src/test/tests.ts

```diff
@@ -606,12 +597,16 @@ describe('Client', function () {
   it('should handle errors that are not an array', function (done) {
     const errorMessage = 'Just an error';
     const payload = {
-      errors: {message: errorMessage},
+      message: errorMessage,
     };
     testBadServer(payload, errorMessage, done);
   });
 
   it('should throw an error when the susbcription times out', function (done) {
+    // TODO
+    // This test is no longer needed. Do u agree Hagaico?
```

Comment:
yep yep alright :) i remembered we removed it but didn't fully looked into this test..